pkg/config/split.go

```diff
@@ -84,7 +84,29 @@ func ConfigureSetSimple(rest string, cfgname string, field reflect.Value) error
 			v := rest == "true"
 			return reflect.ValueOf(&v), nil
 		case reflect.String:
+			var err error
+			rest, err = strconv.Unquote(rest)
```

Comment:
Thios will fail if it is not quoted, which will break anyone who relies on YAML sloppy strings or even contextual interp of numbers as strings.  Maybe not important for these fields, but it IS a breaking change.

see https://github.com/go-delve/delve/pull/3239/files

Reply:
Absolutely, you're right. Was working on some follow-up fixes / refactoring after submitting for testing. They are included in this PR now.